README.md

```diff
@@ -1,4 +1,5 @@
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+[![Build Status](https://travis-ci.org/andela/pippin-ah-backend.svg?branch=develop)](https://travis-ci.org/andela/pippin-ah-backend)
```

Comment:
Please, swap the position of both badges, then when you do that for coveralls as well, put it before the hound badge.

Reply:
Ok. I'm doing it right away

Reply:
Travis CI badge position swapped successfully.